packages/hardhat-core/test/internal/hardhat-network/provider/modules/debug-trace/debug_traceCall.ts

```diff
@@ -0,0 +1,128 @@
+import _ from "lodash";
```

Comment:
For consistency, I would rename this directory from `debug-trace` to just `debug`, and would drop the `debug_` prefix from the test files.

Reply:
I did it now, but I had to add an exception in the .gitignore file because all the "debug" folders are ignored